poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
I'm not gonna write any client functionality in the PR, but my understanding is it needs to work sort of like this: https://www.socketloop.com/tutorials/golang-marshal-and-unmarshal-json-rawmessage-struct-example.  While we probably don't need an explicit `"type"` key, it might be nice to have more that just the status.